dynamics/apis/vendors.py

```diff
@@ -6,6 +6,8 @@ class Vendors(ApiBase):
 
     GET_VENDORS = '/vendors'
     POST_VENDOR = '/vendors'
+    COUNT_VENDOR = "/vendors/$count"
```

Comment:
let's also add support for employees  and locations

Reply:
done